src/sentry/api/endpoints/data_export.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+from sentry import features
+from sentry.api.bases.incident import IncidentPermission
+from sentry.api.base import Endpoint
+from django.utils import timezone
+
+
+from sentry.tasks.data_export import create_record
+
+from sentry.models import ExportedData, Organization
+
+
+class DataExportEndpoint(Endpoint):
+    permission_classes = (IncidentPermission,)
+
+    def get(self, *args, **kwargs):
+        """
+        Retrieve information about the temporary file record.
+        Used to populate page emailed to the user.
+        """
+        organization = Organization.objects.get(slug=kwargs["organization_slug"])
```

Comment:
Since this relates to an `Organization`, this should inherit from `OrganizationEndpoint`. Then you can just define `organization` as a parameter to `get` and it'll be passed automatically.

Reply:
Gave this a shot after refactoring, and it seems it bugs out when i try `def get(self, request, organization, *args, **kwargs)`, but it works if I only include `org` or `request`, any ideas why that is?

Reply:
Not sure what's going on here. Just to confirm, you're inheriting from `OrganizationEndpoint`? Hard to diagnose without seeing code.

You could also check what is actually being passed to request/organization. My assumption is that organization isn't getting passed